doc/customize.md

```diff
@@ -0,0 +1,449 @@
+
+    {
+     ...
+     "rise": {"toolbar_icon": "bar-chart"}
+    }
+
+### Enable a right scroll bar
+
+To enable a right scroll bar when your content exceeds the slide vertical height,
+use the following configuration:
+
+    {
+     ...
+     "rise": {"scroll": true}
+    }
+
+### Usage with Leap Motion
+
+**Reveal.js** supports the [Leap Motion](https://www.leapmotion.com) controller.
+To control RISE slides with the Leap, put the
+[reveal leap plugin options](https://github.com/hakimel/reveal.js#leap-motion)
+in your config with the following parameters:
+
+    {
+     ...
+     "rise": {
+         "leap_motion": {
+            "naturalSwipe"  : true,     # Invert swipe gestures
+            "pointerOpacity": 0.5,      # Set pointer opacity to 0.5
+            "pointerColor"  : "#d80000" # Red pointer"nat.png"
+            }
+        }
+    }
+
+To disable it:
+
+    {
+     ...
+     "rise": {
+         "leap_motion": "none"
+     }
+    }
+
+### `reveal.js` configuration options
+
+`reveal.js` offers a few configuration of its own, [as described in
+`reveal.js`'s documentation](https://github.com/hakimel/reveal.js). Out of this
+category, RISE will pass through the following settings:
+
+* `controls` to enable or disable the left hand side icon buttons
+  (help and exit)
+
+* `progress` to enable or disable the thin progress bar at the bottom
+  of the slideshow
+
+* `slideNumber` that allows you to turn off, or customize, slide
+  numbers. Set to boolean `false` to turn off, [see `reveal.js`'s doc
+  for more details](https://github.com/hakimel/reveal.js#slide-number)
+
+* as well as `history` and `minScale`.
+
+**Note**: The use of the `minScale` option (values other then`1.0`)
+  can cause problems with codemirror.
+
+### Adding custom CSS
+
+RISE looks for two css files to apply CSS changes on top of the
+slideshow view:
+
+* First, it attemps to load `rise.css`, and hence this will be
+applied to all notebooks in the current directory;
+
+* Second, it attempts to load `the_notebook_name.css` and so this will
+hence be only applied to `the_notebook_name.ipynb`.
+
+Both files need to be placed alongside with the notebook of interest,
+i.e. in the same directory.  You can see some examples using this
+customization with `RISE/examples/showflow.ipynb`.
+
+**NOTE.** The implementation of this feature is rather rough, both css
+files are blindly included without checking for their existence, which
+may result in error messages in your browser console, complaining
+about `No such file or directory`. These messages can be safely
+ignored. See also <https://github.com/damianavila/RISE/issues/353> about
+this.
+
+How to customize
+----------------
+
+RISE can be customized in a lot of ways. As of RISE version 5.3, you can:
+
+1. use `nbextensions_configurator`; this tool offers an interactive
+   way to enable, disable and tweak all notebook extensions - see
+   screenshot below;
+
```

Comment:
We should add the `livereveal.json`, in fact, `rise.json` option as well... as I commented in the other PR.

Reply:
yes, right, will be in today's commit